packages/publikator/graphql/resolvers/_mutations/commit.js

```diff
@@ -28,33 +29,61 @@ const {
   }
 } = require('@orbiting/backend-modules-documents')
 
+const {
+  ASSETS_SERVER_BASE_URL
+} = process.env
+
+const generateImageData = async (blob) => {
+  const meta = await sharp(blob).metadata()
+  // image/jpeg -> jpeg
+  // image/png -> png
+  // image/svg+xml -> svg
+  const suffix = blob.type.split('/')[1].split('+')[0]
```

Comment:
got an error on staging
```
failed to transfer image republik-dev/article-test-123-copy-from https://cdn.republik.pink/github/republik-dev/article-das-kartell/images/e9372d1d0ac203f93a59ccc5d62279d1a2a61ae2.jpeg?size=3800x2533#originalURL=images%2Fe9372d1d0ac203f93a59ccc5d62279d1a2a61ae2.jpeg%3Fsize%3D3800x2533 TypeError: Cannot read property 'split' of undefined
```

`blob.type` is specific to `dataUriToBuffer` to I changed it to `meta.format`